OpenMTR/code/ReadData.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.IO;
+using System.Text.RegularExpressions;
+using OpenCvSharp;
+using System.Collections.Generic;
+
+
+namespace OpenMTR
+{
+    static class ReadData
+    {
+        // Get a list of data objects built from all the images found in a directory
+        public static List<DataObject> GetDataObjectList(string DirectoryPath)

```

Comment:
Please adopt the C# coding standards that local variables are camelCased. Class variables are PascalCase, with private class variables starting with an underscore and are camelCased.